Raycasting/src/Geometry.cpp

```diff
@@ -267,7 +267,7 @@ namespace {
 }
 
 
-void Geo::GeoObject::render(const Shader & shader, const VertexBuffer & vb, const IndexBuffer & ib, const VertexArray & va, int inputType)
```

Comment:
Removing const is kinda bad style but I also don't see how this can be avoided when dynamic initialization could be required...

Reply:
This could be avoided by using an initialization function, but that would also break everybody's current code.. I think its fine 😊